src/test/java/org/openjfx/JavaFXBaseMojoTest.java

```diff
@@ -97,6 +112,37 @@ public void invalidPathWithDepthTest() {
         Assert.assertNull(JavaFXBaseMojo.getParent(Paths.get("/some-invalid-path"), 2));
     }
 
+    @Test
+    public void testSplitComplexArgumentString() {
+        String option = "param1 " +
+                "param2   \n   " +
+                "param3\n" +
+                "param4=\"/path/to/my file.log\"   " +
+                "'var\"foo   var\"foo' " +
+                "'var\"foo'   " +
+                "'var\"foo' " +
+                "\"foo'var foo'var\" " +
+                "\"foo'var\" " +
+                "\"foo'var\"";
+
+        String expected = "START," +
+                "param1," +
+                "param2," +
+                "param3," +
+                "param4=\"/path/to/my file.log\"," +
+                "'var\"foo   var\"foo'," +
+                "'var\"foo'," +
+                "'var\"foo'," +
+                "\"foo'var foo'var\"," +
+                "\"foo'var\"," +
+                "\"foo'var\"";
+
+        String splitedOption = new JavaFXRunMojo().splitComplexArgumentStringAdapter(option)
```

Comment:
```suggestion
        String splitedOption = new JavaFXRunMojo().splitComplexArgumentString(option)
```

Because of this, method `JavaFXRunMojo#splitComplexArgumentStringAdapter()` is no longer needed and should be removed.

Reply:
Thanks for the suggestion. Implemented in my commit.